roles/pmm/vars/main.yaml

```diff
@@ -27,6 +27,8 @@ pmm_grafana_alerting_notification: '{{ pmm_grafana_api }}/alertmanager/grafana/c
 
 pmm_setup_alerting: true
 notification_services: ["mysql","mongo","generic","agent","proxysql","postgresql","backup"]
+# Available alert services ["mysql","mongo","generic","agent","proxysql","postgresql","backup"]
```

Comment:
This requires a commented-out example. See [pmm_manage_users](https://github.com/cezmunsta-org/gascan-ansible/blob/b2c78bc258323497997074a3ed3c93b554d80e69/roles/pmm/defaults/main.yaml#L76) as an example

Reply:
This got removed as I reworked this part.